### ISSUE TITLE
How destroy a bulmaCalendar

### ISSUE THREAD

**Issue by leunam90** [Wikiki/bulma-calendar#208]
How can I destroy a bulmaCalendar already initialized

**Comment by drecchia** [comment]
el.bulmaCalendar.destroy(); 
should work

**Comment by daniiy01** [comment]
> el.bulmaCalendar.destroy(); should work

It doesn't work

**Comment by michael-hack** [comment]
I'll take a look in the next days...

**Comment by michael-hack** [comment]
Fixed in 6.1.15

Please note that a previously existing <input> field that was replaced by the calendar will not be restored.

To release the whole object it is necessary to set _el.bulmaCalendar = null;_ after destroy().

**Comment by brendon** [comment]
In case it helps anyone, this is my `stimulus` controller. I found that `destroy()` didn't work as the element was already gone but this seems to work and allows restoration visits via Turbodrive:

```
import 'bulma-calendar/dist/css/bulma-calendar.css'
import bulmaCalendar from 'bulma-calendar/dist/js/bulma-calendar.js'

import { Controller } from '@hotwired/stimulus'

export default class extends Controller {
  static targets = ['dateField']

  connect() {
    const options = {
      type: 'date',
      dateFormat: 'dd/MM/yyyy',
      displayMode: 'dialog',
      showHeader: false,
      color: 'info'
    }

    this.calendar = bulmaCalendar.attach(this.dateFieldTarget, options)[0]
    this.calendarElement = document.getElementById(this.calendar.id)
  }

  disconnect() {
    this.element.appendChild(this.dateFieldTarget)
    this.calendarElement.remove()
    this.calendar = null
  }
}
```
